app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -488,10 +452,7 @@ public void destroy() {
     }
 
     private void createPlayer(int buffer, boolean isInterlaced) {
-        if (mVlcPlayer != null && mIsInterlaced == isInterlaced && mCurrentBuffer == buffer) return; // don't need to re-create
```

Comment:
For this check; `mCurrentBuffer` was never set so this if-statement would never execute the return.